runtime/android/templates/XWalkSdkVersion.template

```diff
@@ -2,4 +2,7 @@ package org.xwalk.core;
 
 class XWalkSdkVersion {
     public static final int SDK_VERSION = ${SDK_VERSION};
+    public static final boolean VERIFY_XWALK_APK = ${VERIFY_XWALK_APK};
+    // This is finterprint of android signing key "Crosswalk Project for Android" in codesign.intel.com
```

Comment:
Typo: "finterprint" instead of "fingerprint". You're missing some "the"s too, and codesign.intel.com is an internal domain so I'd rather not mention it in the source code like that.
Suggestion: _"This is the fingerprint of the "Crosswalk Project for Android" signing key."_